Detailed icon functions and set usages (#3027)
ezsystems/developer-documentation: docs/administration/back_office/back_office_elements/custom_icons.md

    To ensure proper display in the back office, all icons should have SVG format with `symbol`.

Use the scope if you want different icons for different SiteAccesses.

To see more Admin UI's `ids-assets` icons, see [the icon reference](icon_twig_functions.md#icons-reference).

### Access icons in Twig templates

#### Content type icons

> Content type icons are accessible in Twig templates via the `ibexa_content_type_icon` function.

Author: ```suggestion
Content type icons are accessible in Twig templates via the [`ibexa_content_type_icon()` function](icon_twig_functions.md#ibexa_content_type_icon).
```

    To ensure proper display in the back office, all icons should have SVG format with `symbol`.

> Use the scope if you want different icons for different SiteAccesses.

Author: ```suggestion
Use the [scope](multisite_configuration.md#scope) if you want different icons for different SiteAccesses.
```

</svg>
```

#### UI Icons

> See [`ibexa_icon_path`](icon_twig_functions.md#ibexa_icon_path).

Author: ```suggestion
User interface icons are accessible with [`ibexa_icon_path()` function](icon_twig_functions.md#ibexa_icon_path). The function returns a path from an icon identifier and an [icon set](#icon-sets) identifier arguments.
```

                    additional_icons: /assets/images/icons/additional_icons.svg
                default_icon_set: my_icons
```

The icon sets are used by [`ibexa_icon_path()` Twig function](icon_twig_functions.md#ibexa_icon_path).
If you change the `default_icon_set` from one SiteAccess to another, `ibexa_icon_path()` without `set` argument targets icons from different set files.

> If you change the file path of an icon set from one SiteAccess to another, ibexa_icon_path with the same `set` argument targets icons from different set files.

Author: ```suggestion
The icon sets are used by [`ibexa_icon_path()` Twig function](icon_twig_functions.md#ibexa_icon_path).

- If you change the `default_icon_set` from one SiteAccess to another, `ibexa_icon_path(icon)` without `set` argument targets icons from different set files.
- If you change the file path of an icon set from one SiteAccess to another, `ibexa_icon_path(icon, set)` with the same `set` argument targets icons from different set files.
```

                    additional_icons: /assets/images/icons/additional_icons.svg
                default_icon_set: my_icons
```

The icon sets are used by [`ibexa_icon_path()` Twig function](icon_twig_functions.md#ibexa_icon_path).

- If you change the `default_icon_set` from one SiteAccess to another, `ibexa_icon_path(icon)` without `set` argument targets icons from different set files.

> - If you change the file path of an icon set from one SiteAccess to another, `ibexa_icon_path(icon, set)` with the same `set` argument targets icons from different set files.

Author: ```suggestion
- If you change the `default_icon_set` from one SiteAccess to another, `ibexa_icon_path(icon)` without `set` argument targets icons from different set files
- If you change the file path of an icon set from one SiteAccess to another, `ibexa_icon_path(icon, set)` with the same `set` argument targets icons from different set files
```
